magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -279,3 +264,11 @@ dl2_to_dl3:
     source_name: "Crab"
     source_ra: null  # used when the source name cannot be resolved
     source_dec: null  # used when the source name cannot be resolved
+    
+cloud_correction:
+    specify_cloud: yes
+    base_height: 5900
```

Comment:
we could either define it as cm in the config file, or add a comment that it is in cm